frontend/src/main.js

```diff
@@ -1,5 +1,7 @@
 import environment from '~/environment';
 import moment from 'moment'
+import { setupCustomValidationRules } from './components/validation/custom-rules';
```

Comment:
use babel root import for in house modules/components similar with what we have with `environment`